src/CommonITILValidation.php

```diff
@@ -1409,7 +1409,7 @@ public static function dropdownValidator(array $options = [])
         ], [
             'value'               => $type,
             'display_emptychoice' => true,
-            'display'             => $params['display'],
+            'display'             => false,
```

Comment:
I guess `false` should be used instead of `$params['display']` on lines 1426 and 1435.